community/php7/APKBUILD

```diff
@@ -138,214 +136,275 @@ _build() {
+	pkgdesc="$pkgdesc (command line interface)"
 
+	cd "$builddir"
 	make -j1 INSTALL_ROOT="$pkgdir" install || return 1
 
-	install -Dm644 php.ini-production "$pkgdir"/etc/$pkgname/php.ini || return 1
+	ln -sf /usr/bin/php$_suffix "$pkgdir"/usr/bin/php
 
-	local file; for file in pear peardev pecl; do
-		sed -i -e "s~/usr/bin/php~/usr/bin/php7~g" \
-			-e "s~PHP=php~PHP=php7~" \
-			"$pkgdir"/usr/bin/$file || return 1
-	done
-	paxmark -m "$pkgdir"/usr/bin/php7
-
-	find "$pkgdir" -name '.*' | xargs rm -rf || return 1
-	rmdir "$pkgdir"/var/run
+	rm -fr "$pkgdir"/.[[:alpha:]]* || return 1
 }
 
 dev() {
-	default_dev
-	depends="$depends $pkgname $pkgname-pear"
+	provides=$_pkgreal-dev
+	replaces=$provides
+	default_dev || return 1
 
 	mkdir -p "$subpkgdir"/usr/bin \
 		"$subpkgdir"/usr/lib/$pkgname
 
-	mv "$pkgdir"/usr/bin/php-config7 "$pkgdir"/usr/bin/peardev \
-		"$subpkgdir"/usr/bin/ || return 1
-	mv "$pkgdir"/usr/lib/$pkgname/build \
-		"$subpkgdir"/usr/lib/$pkgname/
+	local file
+	for file in php-config phpize; do
+		mv "$pkgdir"/usr/bin/$file$_suffix \
+			"$subpkgdir"/usr/bin/ || return 1
+		ln -sf /usr/bin/$file$_suffix "$subpkgdir"/usr/bin/$file
+	done
+
+	mv "$pkgdir"/usr/lib/$pkgname/build "$subpkgdir"/usr/lib/$pkgname/
 }
 
 doc() {
-	default_doc
-
-	mkdir -p "$subpkgdir"/usr/share/doc/$pkgname
+	provides=
+	replaces=
+	default_doc || return 1
+	install_if="docs $pkgname-config=$pkgver-r$pkgrel"
 
+	cd "$builddir"
 	local file
-	for file in CODING_STANDARDS CREDITS EXTENSIONS INSTALL LICENSE NEWS README* UPGRADING*; do
-		cp "$builddir"/$file "$subpkgdir"/usr/share/doc/$pkgname/ || return 1
+	for file in $(ls CODING_STANDARDS CREDITS EXTENSIONS INSTALL LICENSE NEWS README* UPGRADING*); do
+		install -Dm644 $file "$subpkgdir"/usr/share/doc/$pkgname/$file || return 1
 	done
 }
 
 apache2() {
-	pkgdesc="PHP Module for Apache2"
-	depends="$pkgname-common apache2"
+	provides=$_pkgreal-apache2
+	replaces=
+	pkgdesc="$pkgdesc (apache2 module)"
+	depends="$pkgname-config apache2"
 
-	install -D -m755 "$builddir"/sapi/apache2handler/mod_php7.so \
-		"$subpkgdir"/usr/lib/apache2/mod_php7.so || return 1
+	mkdir -p "$subpkgdir"/usr/lib/apache2
+	cp "$builddir"/sapi/apache2handler/lib$_pkgreal*.so \
+		"$subpkgdir"/usr/lib/apache2/mod_$pkgname.so || return 1
 
-	install -D -m644 "$srcdir"/php7-module.conf \
-		"$subpkgdir"/etc/apache2/conf.d/php7-module.conf
+	install -Dm644 "$srcdir"/$pkgname-module.conf \
+		"$subpkgdir"/etc/apache2/conf.d/$pkgname-module.conf
 }
 
 phpdbg() {
-	pkgdesc="Interactive PHP debugger"
-	depends="$pkgname-common"
+	provides=$_pkgreal-phpdbg
+	replaces=$provides
+	pkgdesc="$pkgdesc (interactive debugger)"
+	depends="$pkgname-config"
 
 	install -Dm755 "$builddir"/sapi/phpdbg/phpdbg \
-		 "$subpkgdir"/usr/bin/phpdbg7
+		 "$subpkgdir"/usr/bin/phpdbg$_suffix || return 1
+	ln -sf /usr/bin/phpdbg$_suffix "$subpkgdir"/usr/bin/phpdbg
 }
 
-embed() {
-	pkgdesc="PHP Embedded Library"
-	depends="$pkgname-common"
+libs() {
```

Comment:
PHP calls it “embed SAPI”, VoidLinux “php-embed”, Fedora “php-embedded”, Arch Linux “php-embed”, Debian “libphp-embed”… So what reason do you have to rename it from php7-embed to php7-libs?!